CHANGELOG.md

```diff
@@ -10,3 +10,7 @@
 ## v1.0.1 (2017-05-23)
 
 * First tagged version.
+
+## v1.0.2 (2021-12-15)
```

Comment:
https://hub.docker.com/r/superbalist/proxysql-exporter/tags

version has moved on a bit, we should update this to `1.1.4` or something.